src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -114,31 +114,6 @@
             <tag name="console.command" command="messenger:stop-workers" />
         </service>
 
-        <service id="console.command.messenger_failed_messages_retry" class="Symfony\Component\Messenger\Command\FailedMessagesRetryCommand">
-            <argument /> <!-- Receiver name -->
-            <argument /> <!-- Receiver locator -->
-            <argument type="service" id="messenger.routable_message_bus" />
-            <argument type="service" id="event_dispatcher" />
-            <argument /> <!-- Retry strategy -->
-            <argument type="service" id="logger" />
-
-            <tag name="console.command" command="messenger:failed:retry" />
-        </service>
-
-        <service id="console.command.messenger_failed_messages_show" class="Symfony\Component\Messenger\Command\FailedMessagesShowCommand">
-            <argument /> <!-- Receiver name -->
-            <argument /> <!-- Receiver locator -->
-
-            <tag name="console.command" command="messenger:failed:show" />
-        </service>
-
-        <service id="console.command.messenger_failed_messages_remove" class="Symfony\Component\Messenger\Command\FailedMessagesRemoveCommand">
-            <argument /> <!-- Receiver name -->
-            <argument /> <!-- Receiver locator -->
-
-            <tag name="console.command" command="messenger:failed:remove" />
-        </service>
-
```

Comment:
Moved into a new file so we can conditionally import that file (instead importing, then removing if the failure transport is disabled).